### PR TITLE
Card and navigation

### PR DESCRIPTION
**Pull request by Sharmaryan**

### CODE REVIEW

**Comment by Sharmaryan** [comment]
Screenshot (35)
Screenshot (36)

File: styles/card/card.css [hunk 1]
```diff
@@ -7,10 +7,12 @@
   margin: 1rem 0;
 }
 .card {
+  border-radius: 1rem;
   color: var(--primary-color);
   width: 16rem;
   padding: 1rem;
   background-color: var(--secondary-color);
+  box-shadow: 3px 3px var(--default);
```
Comment (raghav6202) [inline styles/card/card.css]: can use rem here to maintain uniformity

File: styles/card/card.css [hunk 2]
```diff
@@ -47,7 +49,7 @@
 }
 .card-dismiss,
 .card-text-badge,
-.card-text-badge {
+ .card-text-overlay {
   position: relative;
 }
 .card-dismiss-btn,
```
Comment (raghav6202) [inline styles/card/card.css:55]: missing opening bracket for card-dismiss-btn class

**Review by raghav6202 — APPROVED** [review]
Rest looks good as I had mentioned earlier you can remove the id's if you are not using them. GOOD DESIGN👍

File: styles/card/card.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/card/card.css]: done

File: styles/card/card.css (hunk 2 above)
Comment (Sharmaryan) [reply styles/card/card.css:55]: done

**Comment by Sharmaryan** [comment]
> Rest looks good as I had mentioned earlier you can remove the id's if you are not using them. GOOD DESIGN👍

thanks, and will remove id's later

File: styles/card/card.css [hunk 3]
```diff
@@ -7,10 +7,12 @@
   margin: 1rem 0;
 }
 .card {
+  border-radius: 1rem;
   color: var(--primary-color);
   width: 16rem;
   padding: 1rem;
   background-color: var(--secondary-color);
+  box-shadow: .5rem .5rem var(--default);
```
Comment (LordUttam) [inline styles/card/card.css:15]: --default can have a more descriptive name.

File: styles/card/card.css [hunk 4]
```diff
@@ -12,7 +12,7 @@
   width: 16rem;
   padding: 1rem;
   background-color: var(--secondary-color);
-  box-shadow: 3px 3px var(--default);
+  box-shadow: .5rem .5rem var(--default);
```
Comment (saadahmadhere) [inline styles/card/card.css:15]: values less than 10 px can be used in pixel units.

File: styles/card/card.html [hunk 5]
```diff
@@ -261,7 +261,29 @@ <h3 class="inner-container-types">card with badge</h3>
           src="https://www.thiscodeworks.com/embed/620fb5a114fadf0015661dfd"
           class="code-snippet"
         ></iframe>
+
+
+            <h3 class="inner-container-types">text overlay</h3>
+
```
Comment (saadahmadhere) [inline styles/card/card.html:267]: extra line space here

**Review by saadahmadhere — APPROVED** [review]
Overall code looks clean and well written. Suggested some changes though but it's upto you to implement it.

**Review by LordUttam — CHANGES_REQUESTED** [review]
Why do you have :root in multiple css files? Make one variables.css and have all the variables there with more descriptive names  and import it where you need. 
Rest looks good to me. :)

**Review by jsuryakt — APPROVED** [review]

File: styles/card/card.css (hunk 4 above)
Comment (Sharmaryan) [reply styles/card/card.css:15]: raghav told me to make uniformity use the same unit, i just changed them from px to rem

File: styles/card/card.html (hunk 5 above)
Comment (Sharmaryan) [reply styles/card/card.html:267]: will remove them

File: styles/card/card.css (hunk 3 above)
Comment (Sharmaryan) [reply styles/card/card.css:15]: will refactor them for every component later

**Pull request merged**